src/u2f.c

```diff
@@ -357,10 +416,38 @@ static void _authenticate(const USB_APDU* apdu, Packet* out_packet)
         return;
     }
 
-    if (!workflow_unlock()) {
+    if (!memory_is_initialized()) {
```

Comment:
hmm is this all copy/paste? if so, why not move it into `_handle_unlock()`?

Reply:
I created a `_unlock_or_show_refresh_screen`...